README.md

```diff
@@ -22,11 +22,41 @@ Players need to start Risk of Rain 2, open the console pressing <kbd>CTRL</kbd>
 ```
 connect "<SERVER_IP>:27015";
 ```
-
 Replace `SERVER_IP` with the public IP of the server running the Docker Image.
 
 By default, the server has no password and runs on UDP port 27015. Make sure you have crossplay disabled before joining.
 
+
+## Using Docker Compose 🐋
+
```

Comment:
It would be nice to introduce this section explaining why one would want to use a compose setup instead of a plain docker one, i.e. by stating the advantages (like you did with the description in this PR)